### ISSUE TITLE
safe_user_report

### ISSUE THREAD

**Issue by KKimos** [safe-global/safe-user-allocation-reports#178]
# Related Safe Addresses

Provide a list of Safe addresses that would currently receive a SAFE user allocation.

The list must contain at least 10 eligible Safe addresses at the time of review or it will not be considered. 
The eligible address list can be found here: `safe_user_allocations_reworked.csv` 
Reports will be evaluated on a first-come, first-serve basis.

address for airdrop farmer:

0x3437977766fe4e39d5AdE5206470ae9aeF15f7aC
0xC6dB09a301Bfb0991458074199a905245f13d043
0x4A8C28B424bbd58799b766edeceD5b29F86Cf8fF
0xa0C33d89C9F8782275B4302f0B37818dF4aCC506
0xa8246172595716Fa28D62e0e5E26719A87bA67d1
0x58e940d446C0899179275B53F9eCF77e1C4c65e9
0x02308b53273aB3494D482D5fb3d1F0710ACB0890
0xC18924a3E29045b76327ac6683bBf30dF1c4Ef28
0xa75eE0366af36572C46e2b672dff918e7a624284

0x2A032E7a4575d067953fa85b02a934916b519571
0x299b68AC4725F5Be3063f4Cd33C627A3b12CdcE0
0xcB809e0951054742b1b86761d23B51A56DdaB26F
0xE8D6E069811a736301Ce5EC08ef30A5eaD2EFC6C
0xBFA823f5f7C9A13C0BbF362596Fda6A2b0284Ec8
0x8c5B88Dc02919177452515114a9000E37510cc7C
0x956e68ceCc3096Ca1c1A1a234b8C64404e094446
0xd78ae938f0F6df5156EaB14D60DA23Aa3A4aA6E1
0x85115B8210bE88F28CA35aC38bF272A0E1B8a0af
0x2949353Eb259A91bfb17398DA5451a5Dd8747729

0xaDd69F901bAFf4dA16eDA2c1cA0E966918D33502
0x21fF718dA8F01FdBBA456d85035B9b3529D787aD
0x2916965CB25030CeA3045833E36743bADEF32aeD
0x3bDd8472E1D18A89cD5016789fd1a66bbE352cF2
0x9E77A2839f572f2BF42B294083CAe9Bb4C875264
0x68e77A8145579416f3edA9aEd049d5cf91a53282
0xB6c4a7C7dea089b5eD013Da45a46f4f9605245C7
0xAb345D31C805DfBC6983734010DA19d5EE920058
0x872D65baA9e8fECA7D6dD65796B2855C18C30A54
0xe828fB89DCFdf3f17200aCadC81610Ca6b3bd105

0xabB966b8be466a63Ce3B1c5c1cb84C8d923439d5
0x7C9E2A508d93398B65FA2aecE3F256e3320fb065
0xf5eA4f180458817A597eEF593AE296514a0008e6
0xa1867846ad5A22aF54e0fB30d718a7dD53fcABBb
0xF23A6093E31EB5Ab913c8c459ED97C4D2085824B
0x0968989A87b6FeC3140Ae2FF19EaAC049E48A1D8
0x35e7E80eEdb82e57A89A1E461e53389631F500D0
0x5D37eE15C43fa22E303EbfA94fe7a19941e65fBD
0xdD4dEc45868677D990f98E6Ca823c681a3B0B632
0x217553912f41594E6e1dB45530833E9518F80608

0x2e192Fe2f1D3B41A6c2EF870414701E6154444Da
0xaF67FA1E357A550e78Fb94831ED230500d775020
0x65c5d27FF7facBD993AeEc2ccDdc91174e05c80F
0x0664e2399d15e744893A51f45AA2e855Bffa5e34
0xb3268b45aADC4cC57cd6E723e481520bA9B43a39
0xd28c72C40B6EE8a9b29432ba884E0E2D73295F4d
0x58C43c94F6E85A1A21D8850D273548C2640b7211
0x30664DF7D38b104Dc7a29aeDa3d8ee75afbAb131
0x8b302D87794F62fcE8E8ee1fd1E2Ad7a815D35A8
0x8D0c0dE7B9f6533329AB65658578E607425A9f9f

0x9d2457F5eb132415b8f3c7582791bDAEf0ae77ad
0x2C681dfaeb767b0d2566e60C0f78Bfb7c59c72DB
0x3bED69632A03a987CDEA408788845bD4818844d4
0xfCe853eAF6e6256033c2dE33A0E850838aB0dD4d
0xF7a116312c6cc8aF4e7e1D31E89642B681C14829
0x0167efcf8779540c0c663988223c8aF1C987dD27
0x0e0D46D1Eb35faaf40ec9D6b1EbB20Ed4EBaA6C3
[Related Safe Addresses.pdf](https://github.com/safe-global/safe-user-allocation-reports/files/9530795/Related.Safe.Addresses.pdf)

# Reasoning

These addresses are created in a regular manner, consistent with the following.

1. they are all created out of the same address 0xca33b0edfb55c99d99b1d8f76f07ffe88036083d.

2. they are created very close to each other.

3. they pass into the createProxyWithNonce function in the initializer parameters is exactly the same. They are

   00000006000000000000000000000000ca33b0edfb55c99d99b1d8f76f07ffe8

   8036083d000000000000000000000000290e556cae1d0d17c550edc8f9ed9208

   e268ed5b000000000000000000000000c2a8264a49be2fc5bca778b7a61b1e51

   29169b800000000000000000000000007684370cd8755c4ac5bef0bb29bb7871

   30df681800000000000000000000000043c9b6d9ca494c5e87ecbd524fa7381c

   4f4ae3f4000000000000000000000000a9062f9367987970b6c17128b9ee0cd0

4. after creating safe address, these smart contract transactions are basically not used
   In summary, we have reason to believe that this is a airdrop farmer.

5. These safe addresses are created on a regular basis and usually receive remittances from the same address at one time.

# Methodology

This report was first found in etherscan, where I found that the account is regularly called the createProxyWithNonce function. Then analyzing each transaction, I found that the account was filling in the exact same parameters.

# Safe Address
Please provide a Safe address or ENS name of a Mainnet Safe that will receive any potential rewards earned from this submission.

My safe Address
0x9e19016A90224A864C92Cbe3171436000f9D9723
